### ISSUE TITLE
How to train and test

### ISSUE THREAD

**Issue by hello-peter** [wlin-at/ViTTA#6]
Hello author, what should I do to correctly train and test, I did not find the corresponding training test file. With best wishes

**Comment by wlin-at** [comment]
Hi, thank you so much for the interest in the work. 
Please note that we follow the protocol of test-time adaptation by performing inference right after adapting to a test sample and reporting the accumulated accuracy for all samples. 
The source models Kinetics400 and SSv2 are directly downloaded from the repos of TANet and Video Swin Transformer. The models of UCF101 are trained using the implementation of TANet and Video Swin Transformer without any architecture/training strategy modification. Hyperparameters of training source models are given in the paper. 
The source models are adapted to each test sample and then evaluated on the same sample right after. The `tta_xx.py` scripts output accumulated accuracy for all samples.
